recipes/libftdi/all/conandata.yml

```
@@ -0,0 +1,12 @@
+sources:
+  "1.4":
+    url: "http://developer.intra2net.com/git/?p=libftdi;a=snapshot;h=d5c1622a2ff0c722c0dc59533748489b45774e55;sf=tgz"
+    filename: "libftdi-d5c1622.tar.gz"
```

```suggestion
    url: "https://www.intra2net.com/en/developer/libftdi/download/libftdi1-1.4.tar.bz2"
    sha256: "ec36fb49080f834690c24008328a5ef42d3cf584ef4060f3a35aa4681cb31b74"
```
Checksums of the sources are required for verification.
So I would just use the releases at https://www.intra2net.com/en/developer/libftdi/download.php

---

#### Done

Changed as suggested.